plugins/org.eclipse.fordiac.ide.monitoring/src/org/eclipse/fordiac/ide/monitoring/views/WatchesView.java

```diff
@@ -80,6 +80,7 @@ public class WatchesView extends ViewPart implements ISelectionListener {
 	private final WatchesContentProvider provider = new WatchesContentProvider();
 	private boolean visible = false;
 	private WatchesViewSelectionFilterAction toggleSelection;
+	private boolean refreshingValues = false;
```

Comment:
This variable seems to be never used

Reply:
ah yes this is a left over from an experiment that didn't work will remove it

Reply:
done